releasenotes/notes/civisibility-configure-62d5af67c49e75ad.yaml

```
@@ -0,0 +1,3 @@
+features:
+  - |
+    ci-visibility: Configures the ``CIVisibility`` service to send traces using the ``CIVisibilityWriter`` by default
```

```suggestion
    pytest: Adds the ``DD_CIVISIBILITY_AGENTLESS_ENABLED`` environment variable to configure the ``CIVisibility`` service to use an agent-less test reporting ``CIVisibilityWriter``. Note that the ``CIVisibility`` service will use regular agent reporting by default.
```
Currently ci-visibility is an internal module so I think the scope should be pytest for now since that's the only testing framework we support at the moment. Also as a follow up to the environment variable comment, I was thinking of changing this release note to focus on the option to enable agentless reporting. What do you think?